routes/data.js

```diff
@@ -115,6 +115,48 @@ exports.categoriesGet = async function (req, res) {
   res.send(categories);
 };
 
+exports.memberGet = async function (req, res) {
```

Comment:
?Could we name functions ```{verb}{noun}``` e.g. ```getMember```?

Reply:
Only changed the functions added in this PR to prevent merge conflicts / breaking functionality.